pkg/sql/tenant_settings.go

```diff
@@ -177,6 +177,8 @@ func (p *planner) ShowTenantClusterSetting(
 	}
 
 	name := settings.SettingName(strings.ToLower(n.Name))
+	// NB: We use LookupForLocalAccess since the displayed value is computed in
```

Comment:
Github won't let me comment on it, but I think the line just above this could just be changed from `CheckPrivilege(... VIEWCLUSTERMETADATA)` to `CanManageTenant(...)` and then we don't need any of the other edits in this func nor in the builtin?

Reply:
@rafiss just wanted to check if you had a chance to consider this suggestion?

Reply:
i had missed these comments since it wasn't marked as "changes requested." yes this idea sounds good to me. (though the `setting.InternalKey()`->`setting.Name()` and the usage of `decode_cluster_setting()` changes are both cleanups that i think we should keep.)

btw, not being able to comment on non-edited lines is one of my top gripes with Github reviews as compared to Reviewable.